flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/harness/HarnessTestBase.scala

```diff
@@ -384,4 +386,12 @@ object HarnessTestBase {
       value.row.getField(selectorField).asInstanceOf[T]
     }
   }
+
+  /**
+    * Test class used to test min and max retention time.
+    */
+  class StreamQueryConfigTest(min: Time, max: Time) extends StreamQueryConfig {
```

Comment:
I would rename the class to `TestStreamQueryConfig` because the `Test` at the end suggests that this class is testing something instead of being a util for a test.